common/data_refinery_common/migrations/0007_auto_20190108_2020.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 2.1.2 on 2019-01-08 20:20
+# Modified by Rich
+
+import django.contrib.postgres.fields
+from django.db import migrations, models
+
+def update_cached_values(apps, schema_editor):
```

Comment:
It doesn't look like this is getting called anywhere. I think that you actually have to make a separate migration to do the data migration like the one in this PR: https://github.com/AlexsLemonade/refinebio/pull/783

Basically I think you need one migration to modify the model, and then a second migration to do the data migration because that can't run until after the model has already been modified.